### DISCUSSION TITLE
Minor UI / UX design suggestions

### DISCUSSION THREAD

**Discussion by NovaliX-Dev** [lichess-org/mobile#277] [General]
Here are some of my first feedback regarding the UI :

- Shouldn't the chessboard on the home panel take the whole screen width with the title above and the subtitle below them ? It seems way more logical to me this way

- Imo the text on the play button should be in bold, to make it easier to read

- On the puzzle panel, we have the option to change the number of days in the top-right corner. But it seems it change only the performance chart (the spider's web thing). Therefor it seems more logical to me to put the day selection at the same place, unless they're other things planned i'm not aware of

- There's the "Plus" toggle to show more puzzles categories on the puzzle panel, however the way it's shown confuses me sometimes ;  i think (really) it will hide all the things below (the performance graph and the puzzle history) if i touch it. It makes me think this way because the arrow before the text is facing down when closed and facing up when opened. Imo, the arrow should be facing towards the text when closed, and down when opened, as this indicates in a better way what it contains. Of course, this is a subjective point of view, and i'm open to any discussion regarding this.

**Comment by veloce** [comment]
I agree with the bold play button, but not so much with the rest.

**Comment by NovaliX-Dev** [comment]
Can i have an explanation ? It would help me understand how you see things, so i can try to adapt the suggestions i made and put those who don't fit apart.

I guess the first one would pose problem because the app is also available for tablets where the current layout is more logical.

The third one is problematic because it's planned to add things needing the day selector ?

I understand for the fourth one, as it is too subjective.

**Comment by NovaliX-Dev** [comment]
Maybe i'm too direct with the question, so i want to make clear i understand you don't agree with all my points

**Comment by veloce** [comment]
Here's a more detailed answer because I was in a hurry when I replied first:

- full width chessboard on home is too big imo, if we want to show more chessboard it'd take too much space
- about puzzle panel, you're right, it changes only the dashboard; but the button is conveniently placed in the app bar so it isn't such a big issue imo
- about the plus toggle, it's no issue at all, it's just a matter of convention; for info I'm gonna rework puzzle themes (adding puzzle by openings) and I'm gonna remove the toggle here.

**Comment by NovaliX-Dev** [comment]
> full width chessboard on home is too big imo, if we want to show more chessboard it'd take too much space

On a second though you're probably right about that point, it will look too big.

> about the plus toggle, it's no issue at all, it's just a matter of convention; for info I'm gonna rework puzzle themes (adding puzzle by openings) and I'm gonna remove the toggle here.

I guess this suggestions is resolved then.

> about puzzle panel, you're right, it changes only the dashboard; but the button is conveniently placed in the app bar so it isn't such a big issue imo.

I understand this is not a big issue for you, and it's probably fine that way for most people.

Do you plan to use this day selector for other widgets in that panel ? Like for the history of puzzles ?

**Comment by manyopensource** [comment]
> On a second though you're probably right about that point, it will look too big.

It's not just about it being too big! Why there should be a chessboard when you haven't selected anybody to play with? The place could and should be used to perform the most frequent actions. If you need a ready to play board, you can always go to board analysis.
